unit/java_bytecode/generate_concrete_generic_type/generate_java_generic_type.cpp

```diff
@@ -179,27 +179,27 @@ SCENARIO(
   // We have a 'harness' class who's only purpose is to contain a reference
   // to the generic class so that we can test the specialization of that generic
   // class
-  const irep_idt harness_class=
+  const irep_idt harness_class =
     "java::generic_field_array_instantiation";
 
   // We want to test that the specialized/instantiated class has it's field
   // type updated, so find the specialized class, not the generic class.
-  const irep_idt test_class=
+  const irep_idt test_class =
```

Comment:
does this work with the changed implementation for arrays as generic parameters? These should now be of the form `java::array[reference]of_java::inner_type`